### PR TITLE
add OpenShift route and DNS sections

### PR DESCRIPTION
**Pull request by pgerber**
• document creation of DNS entries
• document creation of routes

### CODE REVIEW

**Comment by pgerber** [comment]
[rendered](https://pgerber-devops-docs.readthedocs.io)

File: openshift/create_nice_installation.rst [hunk 1]
```diff
@@ -79,3 +79,12 @@ Step by Step Instructions
 .. note::
 
   The installation is automatically started once :term:`CD` pushes an image to the Docker registry.
+
+#. Issue an SSL Certificate
+
+    Issue a SSL certificate for ${CUSTOMER}.tocco.ch which is created by the template. See :ref:`issue-ssl-certificate`
+    for instructions.
+
+#. Add additional Routes / Hostnames If Needed
```
Comment (tstjep) [inline openshift/create_nice_installation.rst]: if needed uppercase?

File: openshift/dns.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
+DNS
+===
+
+.. todo::
+
+    * describe how to add new domains (glue entries, etc.)
+
+
+Add DNS Entry (Domains Managed by Us)
```
Comment (tstjep) [inline openshift/dns.rst]: managed/us lowercase

File: openshift/dns.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    .. figure:: dns_static/nine_cname_record.png
+        :scale: 60%
+
+        add record for subdomain (target ``ha-proxy.tocco.ch``)
+
+.. _Nine's Cockpit: https://cockpit.nine.ch/en/dns/domains
+
+
+Add DNS Entry (Domains Managed by a Third Party)
```
Comment (tstjep) [inline openshift/dns.rst]: lowercase ;)

File: openshift/create_nice_installation.rst (hunk 1 above)
Comment (pgerber) [inline openshift/create_nice_installation.rst]: it's a title but I guess you're right about the if

File: openshift/dns.rst (hunk 2 above)
Comment (pgerber) [inline openshift/dns.rst]: looks correct to me considering this is a title

File: openshift/dns.rst (hunk 3 above)
Comment (pgerber) [inline openshift/dns.rst]: ditto

**Comment by pgerber** [comment]
Rebased, just realized I accidentally pushed the "Update Copy Database section for OpenShift" commit to the wrong branch.

Sorry for that

**Pull request merged**